test/integrations/testTypes.ts

```diff
@@ -101,7 +101,7 @@ export type ProcessorTestData = {
       body: ProcessorTransformationResponse[];
     };
   };
-  mockFns?: (mockAdapter: MockAdapter) => {};
+  mockFns?: (mockAdapter: MockAdapter) => void;
```

Comment:
Why are making change here?

Reply:
This changes was suggested by @sanpj2292 while writing the mockFn. Should we revert this?

Reply:
ok then fine

Reply:
ok